### ISSUE TITLE
Last 3 versions not working, from: 0.48.1 to 0.48.3 (Last one)

### ISSUE THREAD

**Issue by Disonantemus** [TeamMsgExtractor/msg-extractor#411]
**Bug Metadata**
* Version of extract_msg: `0.48.1`, `0.48.2`, `0.48.3`
* Your python version:
~~~ocaml
$ pacman -Qi python | head -2
Name            : python
Version         : 3.11.8-1
~~~

* How did you launch extract_msg?
  - [X] My command line or
  - [X] I used the extract_msg package
  
Confused by this, both seems the same to me, details bellow in "How to reproduce" section.

**Describe the bug**
I can install: 0.48.1, 0.48.2, 0.48.3 (Last one) no problem, but when I try to use it, gives error. Last working version installed with pipx is v0.48.

**How to reproduce**
~~~ocaml
$ pipx uninstall "extract-msg"
uninstalled extract-msg! ✨ 🌟 ✨

$ pipx install "extract-msg"
  installed package extract-msg 0.48.3, installed using Python 3.11.8
  These apps are now globally available
    - extract_msg
done! ✨ 🌟 ✨

$ extract_msg --help
Traceback (most recent call last):
  File "/home/kuro/.local/bin/extract_msg", line 8, in <module>
    sys.exit(main())
             ^^^^^^
TypeError: main() missing 1 required positional argument: 'argv'
~~~

**Last working version: 0.48**
~~~ocaml
pipx install extract-msg==0.48
~~~

**Message.msg file to reproduce this**
Not needed.

**Traceback**
Is above ("How to reproduce" section) or maybe I don't know how to get that, no programmer here, I was just trying to use.

**Screenshots**
Not needed.

**Additional context**
~~~ocaml
$ pacman -Qi python-pipx | head -2
Name            : python-pipx
Version         : 1.4.3-1
~~~

---

~~~ocaml
distro: Arch Linux x86_64
kernel: 6.1.52-1-lts
shell: bash 5.2.26
term: tmux
cpu: Intel i7-4790 (8) @ 3.600GHz
gpu: AMD ATI Radeon RX 470/480/570/570X/580/580X/590
~~~

**Comment by TheElementalOfDestruction** [comment]
Ah, I didn't catch this cause I usually run the console script a different way. Took me a bit to figure out that this was caused by the automatically generated console script. This has been fixed in version 0.48.4

**Comment by Disonantemus** [comment]
Last version (0.84.4) now works on my side!
